src/com/SocketCommunication.cpp

```diff
@@ -491,7 +491,7 @@ void SocketCommunication::send(double itemToSend, Rank rankReceiver)
 
 PtrRequest SocketCommunication::aSend(const double &itemToSend, Rank rankReceiver)
 {
-  return aSend(&itemToSend, 1, rankReceiver);
+  return aSend(precice::span<const double>{&itemToSend, 1}, rankReceiver);
```

Comment:
What do the curly braces do? Why `1`?

Reply:
This constructs a span with a single element. I admit that this is ugly.
Maybe something like `precice::wrapInSpan<const double>(*itemToSend)` would be more readable.
Using braces calls the constructor.